2006-05-16  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

	* scripts/check-local-headers.sh: Filter out multiarch headers.

--- glibc-2.4.90/scripts/check-local-headers.sh.multiarch	2006-01-06 13:02:20.000000000 +0100
+++ glibc-2.4.90/scripts/check-local-headers.sh	2006-05-16 01:19:58.000000000 +0200
@@ -30,6 +30,7 @@ if fgrep "$includedir" */*.d |
 fgrep -v "$includedir/asm" |
 fgrep -v "$includedir/linux" |
 fgrep -v "$includedir/selinux" |
+fgrep -v "$includedir/multiarch-" |
 fgrep -v "$includedir/gd"; then
   # If we found a match something is wrong.
   exit 1
